docs/tools/fbsql/fbsql-loaders-kafka.md

```diff
@@ -1,71 +1,41 @@
 ---
-title: fbsql-Kafka
+title: fbsql-loader-kafka
 layout: default
 parent: fbsql
 grand_parent: Tools
 nav_order: 2
 ---
 
-# How can I consume from Kafka using fbsql?
+# Kafka Loader
 
-If fbsql is provided the `--kafka-config=filename` flag, it will run as a Kafka consumer in non-interactive mode. Based on the configuration provided in **filename**, fbsql will read messages from a Kafka topic and submit them to FeatureBase via BULK INSERT statements. In this mode, fbsql processes messages until terminated by the user.
+If fbsql is provided the `--loader-kafka=filename` flag, it will run as a Kafka consumer in non-interactive mode. Based on the configuration provided in **filename**, fbsql will read messages from a Kafka topic and submit them to FeatureBase via `BULK INSERT` statements. In this mode, fbsql processes messages until terminated by the user.
```

Comment:
Where's the flag coming from? Is there a list of valid flags? Add this link under "Before you begin" e.g.,

```
## Before you begin

* [Learn about FBSQL flags](/docs/fbsql/...)
```